go/cmd/declarations/declarations.go

```diff
@@ -75,6 +75,10 @@ func (c *{{.ClassName}}) Merge(other *{{.ClassName}}) {
 		c.values[k] = v
 	}
 }
+
+func (c *{{.ClassName}}) Values() map[string]any {
```

Comment:
why do we want to expose this now? I would prefer to avoid it (or at least make this a private func)

Reply:
removed it